examples/visualisation/get_zarr_coords.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
```

Comment:
is
```
import os
os.environ["L5KIT_DATA_FOLDER"] = "/Users/oliverscheel/L5KIT/data/"
```
needed or should be set anywas?

Reply:
I usually set it in the shell..It's only required to print the dataset tbh. Maybe I'll move the print into the zarr :D